docs/docsite/rst/community/reporting_bugs_and_features.rst

```diff
@@ -25,10 +27,12 @@ When sharing YAML in playbooks, formatting can be preserved by using `code block
 
 For multiple-file content, we encourage use of gist.github.com.  Online pastebin content can expire, so it's nice to have things around for a longer term if they are referenced in a ticket.
 
-If you are not sure if something is a bug yet, you are welcome to ask about something on the mailing list or IRC first.
+If you are not sure if something is a bug yet,i you are welcome to ask about something on the :ref:`mailing list or IRC first <communication>`.
```

Comment:
There's a stray `i` in the middle of this sentence. Call the `i`-herding dogs . . . where's Lexi?